### DISCUSSION TITLE
Thanos receive curl TSDB stat

### DISCUSSION THREAD

**Discussion by BooNny95** [thanos-io/thanos#5668] [Questions & Answers] [answered]
I try to analyze thanos receive local tsdb. Thanos manual tells, that needs to specify THANOS-TENANT header:
https://thanos.io/tip/components/receive.md/#tsdb-stats

Try to run without any header:
```
# curl '0:9092/api/v1/status/tsdb'
404 page not found
```

Then try to set header:
```
# curl -H "THANOS-TENANT: default-tenant" '0:9092/api/v1/status/tsdb'
404 page not found

# curl -H "THANOS-TENANT" '0:9092/api/v1/status/tsdb'
404 page not found

# curl -H "default-tenant" '0:9092/api/v1/status/tsdb'
404 page not found
```

What is the proper way to get tsdb stats for thanos receive?

**Comment by dreiggy** [comment, marked as answer]
Probably you use wrong port. Check the port in flag "--remote-write.address".
